less/index.less

```diff
@@ -55,5 +55,253 @@ table {
 
 html, body {
     height: 100%;
+		// debug passed:
+		// background-color: red;
 }
 
+html {
+	font-family: 'Eczar', serif;
+	font-size: 62.5%;
+
+	// These media adjustments were used to smooth out a few transitions but are now irrelevant.
+	// @media (max-width: 800px) {
+	// font-size: 68.75%;
+	// }
+	// @media (max-width: 500px) {
+	// font-size: 62.5%;
+	// }
+}
+
+// Minimum 1 Color/Font Stack Variable
+
+// Ken's Style: browncoat with a katana
+// pale brown base, steely buckles, flame-red hair
+// Prime = "body" color, Second = "support" color, Tert = "accent" color
+// use shades/tints of main hues for various purposes
+
+@partyPrimeColor: #c06a3e;
```

Comment:
A lot of variables and a lot of comments. This is high level right here. Great job and continue to maintain these habits. I love how you picked hex codes for the variables and not just "grey" or "black" you put in real work on this and it shows.